undefined/instructions.md

```diff
@@ -0,0 +1,7 @@
+## Group Exercise
+
+- Split into groups of 3 or 4.
+- For each example in this section - discuss the code as a group.
+- Why does the example output `undefined`?
```

Comment:
```suggestion
- Why does the output in each example contain `undefined`?
```

Reply:
Just being clear that they may see cases where the output is undefined, but in other cases it might be "Hello undefined" and not just "undefined"